subprojects/enterprise/src/main/java/org/gradle/internal/tasks/impl/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@NonNullApi
+package org.gradle.internal.tasks.impl;
+
+import org.gradle.api.NonNullApi;
```

Comment:
Is there no plugin for this like in GE?

Reply:
What plugin do you speak of?

Reply:
We have a little plugin in GE that generates `package-info.java` files with a similar `@NonNullApi` annotation unless a `package-info.java` file is present.